cgmes/cgmes-extensions/src/main/java/com/powsybl/cgmes/extensions/compatibility/CgmesControlAreasSerDe.java

```diff
@@ -0,0 +1,187 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.cgmes.extensions.compatibility;
+
+import com.google.auto.service.AutoService;
+import com.google.common.base.Strings;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSortedSet;
+import com.powsybl.cgmes.model.CgmesNames;
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.commons.extensions.Extension;
+import com.powsybl.commons.extensions.ExtensionSerDe;
+import com.powsybl.commons.io.DeserializerContext;
+import com.powsybl.commons.io.SerializerContext;
+import com.powsybl.commons.io.TreeDataReader;
+import com.powsybl.iidm.network.*;
+import com.powsybl.iidm.serde.IidmVersion;
+import com.powsybl.iidm.serde.NetworkDeserializerContext;
+import com.powsybl.iidm.serde.TerminalRefSerDe;
+import com.powsybl.iidm.serde.extensions.AbstractVersionableNetworkExtensionSerDe;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.InputStream;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.OptionalDouble;
+
+/**
+ * @author Olivier Perrin {@literal <olivier.perrin at rte-france.com>}
+ */
+@AutoService(ExtensionSerDe.class)
+public class CgmesControlAreasSerDe extends AbstractVersionableNetworkExtensionSerDe<Network, CgmesControlAreasSerDe.DummyExt> {
```

Comment:
why not using `AbstractExtensionSerDe` instead of `AbstractVersionableNetworkExtensionSerDe`. I don't get why we need the versionable part as there's only one the `1.0` version.

Reply:
Yes you're right, there's no need for the versionable management.
I will replace it.

Reply:
Hum no, sorry. I wanted to limit the extension to the IIDM versions <= 1.13.
The extension should not be accepted for the newer versions of IIDM.

Reply:
Hmm, there's no harm in accepting it for versions > 1.13, even though it will never occur, as there won't be any way to get a 1.14 xiidm with such an extension.